config/config.go

```diff
@@ -98,11 +97,11 @@ const (
 	WalletUnlockPasswordFile = "WALLET_UNLOCK_PASSWORD_FILE"
 	// NoOperatorTlsKey is used to start the daemon without using TLS for the operator service
 	NoOperatorTlsKey = "NO_OPERATOR_TLS"
-	// ConnectUrlHost is the host of the tdex-daemon service to connect to ie. myservice:9000
-	ConnectUrlHost = "CONNECT_URL_HOST"
-	// ConnectUrlProto is the http/https protocol of the tdex-daemon service to connect to
-	//used by tdexconnect to create connection url
-	ConnectUrlProto = "CONNECT_URL_PROTO"
+	// ConnectAddrKey is the address <host:port> of the tdex-daemon service to connect to, ie. myservice:9000
```

Comment:
there is a semantic problem I guess: address should include protocol (http,https) meanwhile host is just hostname:address?

Reply:
> [...] meanwhile host is just hostname:address?

i'm sure you were meant hostname:port.. beside this, the naming I use comes from pkg [net/http](https://pkg.go.dev/net/http#Server).. I would not change it just for consistency.